egs/wsj/asr1/cmd.sh

```diff
@@ -10,11 +10,11 @@
 # conf/queue.conf in http://kaldi-asr.org/doc/queue.html for more information,
 # or search for the string 'default_config' in utils/queue.pl or utils/slurm.pl.
 
-export train_cmd="run.pl --mem 2G"
```

Comment:
Same comment
Try to avoid to commit your local changes.
Only `git add` related files.